frontend/src/components/TypeaheadSelect.tsx

```diff
@@ -212,12 +212,10 @@ const TypeaheadSelect: React.FunctionComponent<TypeaheadSelectProps> = ({
   const onInputClick = () => {
     if (!isOpen) {
       openMenu();
-      setTimeout(() => {
-        textInputRef.current?.focus();
-      }, 100);
-    } else if (isFiltering) {
-      closeMenu();
     }
+    setTimeout(() => {
+      textInputRef.current?.focus();
+    }, 100);
   };
```

Comment:
This change seems reasonable to me and does fix the focus issue, but I'm curious if @jeff-phillips-18 has an opinion on it since he wrote this code in https://github.com/opendatahub-io/odh-dashboard/pull/3094.

Reply:
Looking at this, I would say it is ok to remove the timeout, but in the `<Select>` component below, add parameter:
```
shouldFocusFirstItemOnOpen={!selected}
```

This will keep focus on the input when the user already has something selected.

Reply:
@jeff-phillips-18 what about the removal of the `closeMenu()` call here? I don't think I'm seeing the menu close when clicking the input anyway, I wonder if that's getting counteracted.